test/terra/backends/qasm_simulator/qasm_thread_management.py

```diff
@@ -102,7 +102,13 @@ def test_parallel_thread_defaults(self):
         """Test parallel thread assignment defaults"""
 
         opts = self.BACKEND_OPTS
-        max_threads = multiprocessing.cpu_count()
+        
+        # Run single circuit, no noise to get max_threads
+        result = execute(self.dummy_circuit(1),
```

Comment:
This could be made a class function so you don't have to repeat it
```suggestion
def available_threads(self):
    """"Return the threads reported by the simulator"""
    result = execute(self.dummy_circuit(1),
                             self.SIMULATOR,
                             shots=1,
                             backend_options=self.BACKEND_OPTS).result()
    return self.threads_used(result)[0]['total']
```